lineapy/transformer/node_transformer.py

```diff
@@ -152,6 +152,16 @@ def generic_visit(self, node: ast.AST):
             node,
             ast.stmt,
         ):
+            if (
+                isinstance(node, ast.FunctionDef)
+                and len(node.decorator_list) > 0
+            ):
+                min_decorator_line = min(
+                    [decorator.lineno for decorator in node.decorator_list]
+                )
+                # this might not be needed but adding in case older python has weirdness
+                if min_decorator_line is not None:
+                    node.lineno = min(min_decorator_line, node.lineno)
```

Comment:
Wow nice find!!! Im surprised that this bug was platform dependent. So weird.

Reply:
it wasnt.. it failed for me too that was the hint. it wasnt as extreme as your case which failed entirely.